### ISSUE TITLE
Issues installing with python3.6

### ISSUE THREAD

**Issue by ekohilas** [carpedm20/fbchat#204]
Installing using pip --user and running the example code fails like so

```
Traceback (most recent call last):
  File "message.py", line 7, in <module>
    client = Client(username, password)
  File "/home/user/.local/lib/python3.6/site-packages/fbchat/client.py", line 74, in __init__
    self.login(email, password, max_tries)
  File "/home/user/.local/lib/python3.6/site-packages/fbchat/client.py", line 382, in login
    login_successful, login_url = self._login()
  File "/home/user/.local/lib/python3.6/site-packages/fbchat/client.py", line 243, in _login
    soup = bs(self._get(self.req_url.MOBILE).text, "lxml")
  File "/home/user/.local/lib/python3.6/site-packages/bs4/__init__.py", line 165, in __init__
    % ",".join(features))
bs4.FeatureNotFound: Couldn't find a tree builder with the features you requested: lxml. Do you need to install a parser library?
```

I had installed bs4 previously
and running
```
python3.6 -m pip install lxml --user
```
gives
```
Requirement already satisfied: lxml in /usr/lib/python3/dist-packages
```
instructions on [Stack Overflow](https://stackoverflow.com/questions/17766725/how-to-re-install-lxml) note that fixing this requires re-installing lxml.

However doing so gave this error.
```
  File "/usr/lib/python3.6/re.py", line 142, in <module>
    class RegexFlag(enum.IntFlag):
AttributeError: module 'enum' has no attribute 'IntFlag'
```
This was because of [this](https://stackoverflow.com/a/45716067) issue

From what I understand, the pip setup needs to be updated such that the enum package is not installed under python3.6

**Comment by madsmtm** [comment]
You're probably right, but the problem is that to maintain python 2.7 support, we need `enum34`. Maybe you know how to only make it a requirement on python 2.7?

**Comment by ekohilas** [comment]
I've made a change that I will be pulling later :)

**Comment by ekohilas** [comment]
The change I've made hasn't updated anything.
Where/How is the requires.txt file generated?

**Comment by madsmtm** [comment]
I'm sorry, when building the pip distribution, i forgot to clean my build dir. Should be fixed in `1.0.22`, please verify

**Comment by ekohilas** [comment]
@madsmtm 
This was found in the requires.txt file
```
requests
lxml
beautifulsoup4

[: python_version == "2.7" ]
enum34
```
causing the build to break

This needs to be updated to
```
requests
lxml
beautifulsoup4
enum34; python_version == '2.7'
```

Additionally, if the project should be compatible with versions before 3.4, then it is better written as `enum34; python_version < '3.4'` instead. (and respectively changed in requirements.txt)

This #207 has the same issue.

**Comment by madsmtm** [comment]
You're right, I assumed pip copied the file but it was generating the `requires.txt` file in a wierd way. Should be fixed in `1.0.23`.
I'll change it to `enum34; python_version < '3.4'` in v. `1.0.24`, but supporting python3 versions below 3.4 is really not a concern of mine (Those version aren't even supported by the python community)

**Comment by ekohilas** [comment]
Fair enough, I was thinking the same when I first made the commit.
I've also confirmed that the conditional requirement is working as expected.
